src/libaktualizr/utilities/dequeue_buffer.cc

```diff
@@ -23,8 +23,8 @@ void DequeueBuffer::Consume(size_t bytes) {
     throw std::logic_error("Attempt to DequeueBuffer::Consume() more bytes than are valid");
   }
   // Shuffle up the buffer
-  auto next_unconsumed_byte = buffer_.begin() + bytes;
-  auto end_of_written_area = buffer_.begin() + written_bytes_;
+  auto* next_unconsumed_byte = buffer_.begin() + bytes;
+  auto* end_of_written_area = buffer_.begin() + written_bytes_;
```

Comment:
Is this to prevent the variables from becoming arrays or to prevent them from being the wrong kind of const?

Reply:
It's only for readability: https://clang.llvm.org/extra/clang-tidy/checks/readability-qualified-auto.html